docs/3.0-migration-guide.md

```diff
@@ -725,7 +725,7 @@ If you prefer a different order, call `mbedtls_ssl_conf_curves()` when configuri
 ### SSL key export interface change
 
 This affects users of the SSL key export APIs:
-```
+```C
```

Comment:
Minor: This block isn't actually in C syntax. This probably shouldn't have been a block at all.